cogs5e/models/character.py

```diff
@@ -479,6 +536,15 @@ def get_sheet_embed(self):
                                   f"Please run !update.")
 
         return embed
+    
+    def active_context(self,ctx):
+        #retrieves which contexts the character is active in
+        contexts=[]
+        if self._active:
+            contexts.append("GLOBAL")
+        if (str(ctx.guild.id) if ctx.guild is not None else None) in self.active_guilds:
+            contexts.append("SERVER")
+        return contexts
```

Comment:
I'm not a huge fan of this method since it makes assumptions about how this data is going to be displayed - consider splitting this into two methods `is_global_active() -> bool` and `is_server_active(ctx) -> bool`